release-notes/cloudflow-1.3.1.md

```diff
@@ -0,0 +1,42 @@
+# Cloudflow 1.3.1 Release Notes
+
+Cloudflow 1.3.1 is a patch release and contains the following changes:
+
+- Upgraded to Spark 2.4.5 and Spark Operator 0.6.7
+- Upgraded to Flink 1.10 and Flink Operator 0.8.2
+- Upgraded to Kafka 2.4.0 and Strimzi operator 0.16.2
+- Upgraded to Lightbend Console 1.2.9 (Enterprise edition only)
+- Upgraded to include Alpakka Kafka 2.0.2, which includes significant speed ups. Also updated the Akka streams Streamlet APIs to match.
+- Streamlet HTTP endpoints now get predictable port numbers instead of randomized ones
+- Support for fully uninstalling Cloudflow including all associated resources, dependencies and deployed applications
+- The CLI now supports Cloudflow installations in non-default namespaces by auto-detecting the installation namespace
+- The OSS installer now supports Amazon Elastic Kubernetes Service (EKS)
+
+**Known issues with this release:**
+
+- The Enterprise edition cannot be installed on K8s version 1.16 or higher due to a compatibility issue with the Lightbend Console.
+- The `kubectl cloudflow status` command does not report a correct rollup status for the entire application when it includes Flink streamlets; e.g. it reports `Pending` even though all individual streamlets are `Running`.
```

Comment:
May be we can mention that this does not affect the running of the application in any way

Reply:
Good point